src/services/utilities.ts

```diff
@@ -906,6 +906,20 @@ namespace ts {
         return isTemplateLiteralKind(token.kind) && position > token.getStart(sourceFile);
     }
 
+    export function isInJSXText(sourceFile: SourceFile, position: number) {
+        const token = getTokenAtPosition(sourceFile, position);
+        if (isJsxText(token)) {
+            return position >= token.getStart(sourceFile);

```

Comment:
You probably want to use `token.pos` because `getStart` will handle parts of the JsxText as trivia.

Reply:
on second thought: is this even necessary? JsxText cannot have leading trivia, therefore you can always `return true;` here